template-only-bin/destroy-account.sh

```diff
@@ -17,6 +17,7 @@ cd infra/accounts
 sed -i.bak 's/resource "aws_s3_bucket" "tf_state" {/&\n  force_destroy = true/' ../modules/terraform-backend-s3/main.tf
 sed -i.bak 's/resource "aws_s3_bucket" "tf_log" {/&\n  force_destroy = true/' ../modules/terraform-backend-s3/main.tf
 sed -i.bak 's/prevent_destroy = true/prevent_destroy = false/g' ../modules/terraform-backend-s3/main.tf
+sed -i.bak 's/resource "aws_s3_bucket" "load_balancer_logs" {/&\n  force_destroy = true/' ../modules/service/main.tf
```

Comment:
this won't do anything since destroy-account.sh gets called after the service is already destroyed. you probably meant to update `template-only-bin/destroy-app-service.sh`?

Reply:
Moved it over and added apply